com.unity.render-pipelines.high-definition/Documentation~/deep-learning-super-sampling-in-hdrp.md

```diff
@@ -62,6 +62,22 @@ Your Unity project now supports DLSS and you can now enable DLSS for Cameras in
 The **Use Optimal Settings** checkbox in the [HDRP Assets](HDRP-Asset.md) is enabled by default. This means that DLSS sets the dynamic resolution scale automatically.
 If you disable this checkbox DLSS uses the same dynamic resolution scale set by the project. For more information see the [Dynamic Resolution](Dynamic-Resolution.md) guide.
 
+### Mip bias in DLSS
+
+To enable automatic mip bias correction when DLSS is enabled, enable the checkbox **Use Mip Bias** in the [HDRP Asset](HDRO-Asset.md).
+
+If a specific custom mip bias is desired for a texture, the following custom function node can be created with the following body:
+
```

Comment:
To enable automatic mip bias correction when you enable DLSS, open the  [HDRP Asset](HDRP-Asset.md) and enable the **Use Mip Bias** checkbox .

If you need a specific custom mip bias for a Texture, create a custom sampler that samples from TextureInput, SamplerInput, UV, and MipBias. To do this, enter the following script into the **Node Settings** ' **Body** field. The images below display this example:

Reply:
Done, thanks Vic!